my_set/main/urls.py

```diff
@@ -3,7 +3,10 @@
 
 urlpatterns = [
     path('', views.index, name='home'),
-    path('register/', views.registerPage, name='register'),
-    path('login/', views.loginPage, name='login'),
-    path('logout/', views.loginPage, name='logout'),
+    path('project_list/', views.project_filter_view, name='project_list'),
+    path('register/', views.register_page, name='register'),
+    path('login/', views.login_page, name='login'),
+    path('logout/', views.logout_user, name='logout'),
+    path('import_csv/', views.import_csv_view, name='import_csv'),
+    path('add-project/', views.add_project, name="add_project"),
```

Comment:
Endpoint urls should inlcude dash (`-`) rather then underscore (`_`) as a separator between words.